.env

```
@@ -0,0 +1,15 @@
+# FinTrac Environment Configuration
+VITE_USE_POUCHDB=false
+VITE_SYNC_ENABLED=true
+VITE_COUCHDB_URL=http://10.196.88.186:5984
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_COUCHDB_URL key should go before the VITE_SYNC_ENABLED key

(UnorderedKey)